ruby-monday-blog/app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,7 @@
 class PostsController < ApplicationController
 
   def index
-    @posts = Post.ordered_by_created_at
+    @posts = Post.ordered_by_created_at.includes(:tags)
```

Comment:
This addresses the N+1 query problem present in the views.  Here's a screencap of the more efficient SQL query:

![image](https://cloud.githubusercontent.com/assets/7947900/9203993/0e8c29d6-4020-11e5-998b-7fb5339dcc20.png)